infoschema/tables.go

```diff
@@ -66,6 +66,7 @@ const (
 	tableTableSpaces                        = "TABLESPACES"
 	tableCollationCharacterSetApplicability = "COLLATION_CHARACTER_SET_APPLICABILITY"
 	tableProcesslist                        = "PROCESSLIST"
+	tableTiDBIndexes                        = "TIDB_INDEXES"
```

Comment:
Directly use `INDICES` as table name?

Reply:
better not, index is nore standard sql concept, for example: mysql use `innodb_indexes`